### ISSUE TITLE
[READ THIS] Anyone new and old coming here in search for fixes!

### ISSUE THREAD

**Issue by QiuIcs** [PopcornTimeTV/PopcornTimeTV#910]
This version of PopocornTime has been **TERMINATED**. It is no longer developed.
If you want PopcornTime on your iOS device (works on iOS14 beta1) or AppleTV get it from [here](https://antiquebeta.github.io/getpopcorn.app/). 
[Antique_Dev](https://twitter.com/Antique_Dev) is maintaining it now, and you should follow his twitter for updates.

Also you can donate to his [Patreon](https://www.patreon.com/getpopcorn) to help keep this updated.

The new app looks super nice and slick and everything is working now.

Cheers!

**Comment by PencilNotPen** [comment]
Does this version have plans for a tvOS release?

**Comment by QiuIcs** [comment]
@PencilNotPen already has one. But it's still in development. You can acces the beta by joining the patreon members.

**Comment by swingchampion** [comment]
**ATTENTION**. Do not use this app. The whole project is strangely not OpenSource and nobody knows what the app really does in the background. Furthermore the developer demands money for the tvOS app. I don't think it's okay to take an OpenSource App and then charge money for it. Please **close** thsi issue.

**Comment by QiuIcs** [comment]
@swingchampion dude take a chill pill. Devs need motivation to work on shit, it can't all be free. He did this on his own, and the fact that he "charges" money is to help future development. Plus the tvOS app is still in beta stages and that's why only patrons can have it, because "they understand what beta means" and it may have more bugs. Do you know how Patreon works?

If you have doubts you can talk with Antique_Dev on twitter, and I'me sure he'll be more than happy to assist on you questions.

Stop making asumptions just because you don't know what's going on or you think he may steal your info.

**Comment by Midifreakz** [comment]
I understand that a dev wants some bucks but 'buymeacoffee" would have been a better option. I think it's better to keep PT on github. I also don't like the closed nature of the project.

**Comment by QiuIcs** [comment]
Reach the dev on twitter and ask away. Maybe when it’s more complete he will open source it.

**Comment by sqig** [comment]
How do you download the tvOS. version from the link the OP ? Has anyone tried this on an  Apple TV4K. ?
Thank you.

**Comment by QiuIcs** [comment]
@sqig the tvOS app is not out yet, still beeing worked on. You can acces the curent beta version by joining his Patreon.

**Comment by sweetw0r** [comment]
What happened to the maintainers of the current repo? How’s their well being? Can’t they update the READ.ME with some official statements like they did few years back before the repo was reborn from the ashes again?

**Comment by OKNoah** [comment]
Yeah, I don't trust anyone randomly repo-jacking and straight asking for money for access to an open source project.

**Comment by proellusionist** [comment]
> Yeah, I don't trust anyone randomly repo-jacking and straight asking for money for access to an open source project.

Well no - it’s not repo jacking, 90%+ of the code is his own work from scratch. The only bit he uses from the repo is PopcornKit. And yes I do know this, despite it being closed source, because I have seen him coding it live, and so have others.

One main reason for the closed source nature is to stop people outright cloning what is essentially 90%+ his own work and just injecting malware and adverts in it. 

The apps are free and without any advertising in them for EVERYONE. For Patrons they get to support the dev and have input on the features and design, and get the betas early.  And no he is not “straight up asking for money for an open source project” because the open source iOS app doesn’t work on iOS 13 and iOS 14, even if there was a fork that did, I prefer this one cause it’s well crafted and hard work put into it to maintain it. 

Also anyone with a bit of a brain can track the network usage and see he is not “stealing your data” or whatever you think being closed source means.

So take a chill pill cause no one is demanding you pay anything or even use the app. In fact there was no Patreon at all, until some of us users suggested it to him because we loved the work he was doing and wanted to support him.

**Comment by sweetw0r** [comment]
I think the guy is well known in certain circles. Used to code for iOS jailbreak projects

**Comment by PlasmaSoftUK** [comment]
I will be staying well clear of any "paid for" closed source versions of this. It's been free and community maintained for years. There is even a pull request on here which adds 3 lines to one file and it then works fine on tvOS 14. Many of the forks are still being maintained as well so all is not lost.

**Comment by proellusionist** [comment]
> I will be staying well clear of any "paid for" closed source versions of this. It's been free and community maintained for years. There is even a pull request on here which adds 3 lines to one file and it then works fine on tvOS 14. Many of the forks are still being maintained as well so all is not lost.

Antique is a trusted jailbreak tweak developer and the app is not “paid for” if you are referring to his version (which is 95%+ his own code)

**Comment by PlasmaSoftUK** [comment]
> the app is not “paid for” if you are referring to his version, it is free (the app is also 95%+ his own code)

According to comments above if you want the tvOS version it's currently hidden behind a Patreon pay wall, which is what I was referring to. Don't get me wrong I don't expect anyone to work for free, but if the app is truly free then the payments/donations should be voluntary and not forced.

**Comment by proellusionist** [comment]
> > the app is not “paid for” if you are referring to his version, it is free (the app is also 95%+ his own code)
> 
> According to comments above if you want the tvOS version it's currently hidden behind a Patreon pay wall, which is what I was referring to. Don't get me wrong I don't expect anyone to work for free, but if the app is truly free then the payments/donations should be voluntary and not forced.

Then you have misunderstood, the tvOS version is not “behind a paywall”, it’s just early access for patrons, once it’s a bit more polished up (it’s not near the quality the iOS version is as of yet) then it will be opened up to everyone just like the iOS version. “Paywall” presumes that you have no other option but to pay for it, which is not the case, all people have to do is just wait a bit longer for new updates.

**Comment by sedwo** [comment]
hmm.. forgive my suspicion but when you point us to a landing page that has its `Contact`, `About`, `FAQ's` and `Support` (plus its social media icons) links all point to being rick-rolled (https://www.youtube.com/watch?v=dQw4w9WgXcQ), it kind of triggers my trust issues.

This open-source repo is quite active still and provides a much greater learning opportunity for the community then a closed-source package.  Let's keep the education open here, just like the team at Kodi does, for all to benefit and learn from.

**Comment by antiquebeta** [comment]
@sedwo the reason for the rick rolls is because it's funny and for now I don't have those pages completed. You'll also notice an aggressive front for the website and Twitter (also funny).

I've open sourced the base project I created before working on top of it, it is all my own process and anything built on top of the base is also my own code, it can be found [here](https://github.com/getpopcorn/Popcorn)

I'll no longer be replying to any comments here or otherwise, I will however be fixing broken code and issues in the original when I'm free. Could take some time though.

**Comment by OKNoah** [comment]
@QiuIcs @antiquebeta I’ve never seen behavioral so antithetical to the values of open source and communities like Github. You have absolutely no business here.

**Comment by proellusionist** [comment]
@OKNoah LOL, so dramatic!

**Comment by Ioan-Popovici** [comment]
> @QiuIcs @antiquebeta I’ve never seen behavioral so antithetical to the values of open source and communities like Github. You have absolutely no business here.

I work on open-source... It doesn't pay. Buy me coffee does not work, it only hurts the project.
A healthy project must have a source of income otherwise there is little incentive to go on.
I don't see anyone else picking up this project and fix the long-standing bugs.
There hasn't been progress on this project for years now, what other solution do you propose?
